### ISSUE TITLE
Pagination / next / previous for multiple slides in viewport

### ISSUE THREAD

**Issue by ivancuric** [metafizzy/flickity#86]
Currently the pagination and previous/next buttons work as though there was a single slide visible in the viewport. What happens is that with multiple slides visible you get pagination skipping and with keyboard control, you get no movement.

The problem is that both only switch the `.is-selected` slide to the `cellAlign` value.

Ideally flickity should have an option to detect the number of slides currently in the viewport, split all the slides into equally-sized groups relative to the `cellAlign` option which would either overlap for per-slide arrow/keyboard control, or be discrete for page-by-page navigation.

Basically if there are 5 slides visible out of  8 total, you should be able to have these 2 options, depending on the type of use case:
* 4 screens: `[12345]` `[23456]` `[34567]` `[45678]`
* 2 screens: `[12345]` `[678]`

**Comment by desandro** [comment]
Thanks for this feature request. Sounds like a duplicate of "Slide by group" #22. Closing as a duplicate.
